sed/io.py

```diff
@@ -1,10 +1,44 @@
+from typing import Union
+
 import h5py
-import xarray as xr
 import numpy as np
+import xarray as xr
+
+DEFAULT_UNITS = {
+    "X": "step",
+    "Y": "step",
+    "t": "step",
+    "tofVoltage": "V",
+    "extractorVoltage": "V",
+    "extractorCurrent": "A",
+    "cryoTemperature": "K",
+    "sampleTemperature": "K",
+    "dldTimeBinSize": "ns",
+    "delay": "ps",
+    "timeStamp": "s",
+    "energy": "eV",
+    "kx": "1/A",
+    "ky": "1/A",
+}
+
+
+def recursive_write_metadata(h5group: h5py.Group, node: dict):
+    """
+    Recurces through a python dictionary and writes it into an hdf5 file.
```

Comment:
Docstring starts in previous line, the one with the quotes :)

Reply:
So what are the linters good for, if they don't fix this ;)

Reply:
Linters only do "cosmetic" fixes. They are still not smart enough :)